app/controllers/carts_controller.rb

```diff
@@ -14,7 +14,27 @@ def checkout_cart
       flash[:alert] = "You must login to checkout"
       redirect_to showcart_path
     else
-      redirect_to create_order_path
+      cart = Cart.new(session[:cart])
+      total_cost = 0
+      cart.projects.each do |project, quantity|
+        total_cost += project.price * quantity.to_i
+      end
+
+      @order = Order.create(
+        user_id: session[:user_id],
+        total_cost: total_cost,
+        status: "ordered"
+        )
+      cart.projects.each do |project, quantity|
+        Loan.create(
+          project_id: project.id,
+          order_id: @order.id,
+          quantity: quantity,
+          line_item_cost: project.price * quantity.to_i
+          )
+      end
+      session.delete(:cart)
+      redirect_to user_order_path(user_id: session[:user_id], id: @order.id)
```

Comment:
So I have a question about the usage of quantity in the Cart. Orion and I changed the quantity to be loan_amount because 1) we no longer need quantity in the cart and 2) the more appropriate data we need now is the loan amount. Lenders can no longer "purchase" 2 loans so it doesn't make sense to have quantity. Orion and I made these changes in our PR so perhaps it's best that we merge yours and look for conflicts?